norfair/camera_motion.py

```diff
@@ -211,8 +211,17 @@ def __init__(
         self.proportion_points_used_threshold = proportion_points_used_threshold
 
     def __call__(
-        self, curr_pts: np.ndarray, prev_pts: np.ndarray
-    ) -> Tuple[bool, HomographyTransformation]:
+        self, curr_pts: Optional[np.ndarray], prev_pts: Optional[np.ndarray]
+    ) -> Tuple[bool, Optional[HomographyTransformation]]:
+
+        if prev_pts is None or curr_pts is None or prev_pts.shape[0] < 4:
+            # Preventing the following error: cv2.error: OpenCV(4.8.0)
+            # /io/opencv/modules/calib3d/src/fundam.cpp:385:
+            # error: (-28:Unknown error code -28) The input arrays should have
+            # at least 4 corresponding point sets to calculate Homography
+            # in function 'findHomography'
+            return True, None
```

Comment:
Instead of returning `None` for the transformation when it couldn't be updated, I would prefer to return `HomographyTransformation(self.data)` so that at least the `MotionEstimator` still returns the coordinate transformation corresponding to the previous frame. So the last line should be

`return True, HomographyTransformation(self.data)`

Also, I would warn the user that the homography couldn't be computed in this frame due to low amount of points. Just add  `from logging import warning` at the top of this file and a warning message here.